app/src/main/java/com/randomappsinc/studentpicker/database/DataSource.java

```diff
@@ -134,6 +134,38 @@ public void renameList(int listId, String newName) {
         close();
     }
 
+    public Map<ListDO, List<NameDO>> getAllListSet() {
```

Comment:
Having a Map to represent a relationship between 2 classes seems hacky. I think we should just modify ListDO to have a list of NameDO.